client/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
Do you intend for robots to visit the site?, if not `Disallow: /` is the way to go

Reply:
No, missed checking this when `create-react-app` generated its code.